common/src/test/java/com/google/auto/common/MoreTypesIsTypeOfTest.java

```diff
@@ -15,198 +15,171 @@
  */
 package com.google.auto.common;
 
-import static com.google.common.collect.Iterables.getOnlyElement;
 import static com.google.common.truth.Truth.assertWithMessage;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
-import com.google.common.collect.Iterables;
+import com.google.common.collect.ImmutableList;
 import com.google.testing.compile.CompilationRule;
-import javax.lang.model.element.Element;
+import java.util.List;
+import java.util.SortedMap;
 import javax.lang.model.element.TypeElement;
+import javax.lang.model.type.TypeKind;
 import javax.lang.model.type.TypeMirror;
 import javax.lang.model.util.Elements;
+import javax.lang.model.util.Types;
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-/**
- * Tests {@link MoreTypes#isTypeOf}.
- */
+/** Tests {@link MoreTypes#isTypeOf(Class, TypeMirror)}. */
 @RunWith(JUnit4.class)
 public class MoreTypesIsTypeOfTest {
 
   @Rule public CompilationRule compilationRule = new CompilationRule();
 
-  private Elements elements;
+  private Elements eltUtils;
```

Comment:
Would you mind spelling this as `elementUtils`? The abbreviation is a bit disconcerting.

Reply:
No problem. Would you like me to revert it back to `elements`, as it was before?

Regarding the `eltUtils`, I just want to mention that this is the name used in the official java documents; for example, [here](https://docs.oracle.com/javase/7/docs/api/javax/annotation/processing/Filer.html).

Reply:
Thank you for your thorough answers, I really appreciate it. Done.